src/GovUk.Education.ExploreEducationStatistics.Content.Services.Tests/DataSetServiceTests.cs

```diff
@@ -0,0 +1,205 @@
+using System;
+using System.Threading.Tasks;
+using GovUk.Education.ExploreEducationStatistics.Common.Tests.Extensions;
+using GovUk.Education.ExploreEducationStatistics.Common.Tests.Fixtures;
+using GovUk.Education.ExploreEducationStatistics.Content.Model.Database;
+using GovUk.Education.ExploreEducationStatistics.Content.Model.Extensions;
+using GovUk.Education.ExploreEducationStatistics.Content.Model.Repository;
+using GovUk.Education.ExploreEducationStatistics.Content.Model.Tests.Fixtures;
+using Xunit;
+using static GovUk.Education.ExploreEducationStatistics.Content.Model.Tests.Utils.ContentDbUtils;
+
+namespace GovUk.Education.ExploreEducationStatistics.Content.Services.Tests;
+
+public class DataSetServiceTests
+{
+    public class GetDataSetTests : DataSetServiceTests
+    {
+        private readonly DataFixture _fixture = new();
+
+        [Fact]
+        public async Task Success()
+        {
+            var publication = _fixture.DefaultPublication()
+                .WithReleaseParents(
+                    _fixture.DefaultReleaseParent(publishedVersions: 1, draftVersion: true,  year: 2000)
+                        .Generate(2))
+                .WithTopic(_fixture.DefaultTopic()
+                    .WithTheme(_fixture.DefaultTheme()))
+                .Generate();
+
+            var releaseFile = _fixture.DefaultReleaseFile()
+                .WithRelease(publication.Releases[0])
+                .WithFile(_fixture.DefaultFile())
+                .Generate();
+
+            var contentDbContextId = Guid.NewGuid().ToString();
+            await using (var contentDbContext = InMemoryContentDbContext(contentDbContextId))
+            {
+                await contentDbContext.ReleaseFiles.AddRangeAsync(releaseFile);
```

Comment:
[Styleguide 2.3.2](https://github.com/dfe-analytical-services/explore-education-statistics/blob/dev/styleguides/styleguide-backend.md#232-entity-framework-ef) - Use `contentDbContext.ReleaseFiles.Add(releaseFile);` rather than the async variant.

Applies to multiple occurrences in this file.

Could also consider making this `contentDbContext.Publications.Add(publication);` Feels more natural to be saving the graph top-down but I don't think it matters at all.